RDMMonitor.js

```diff
@@ -318,56 +309,62 @@ async function UpdateInstance(instance)
 function UpdateDevices()
 {
     return new Promise(function(resolve) {
-        request.get(config.url+DEVICE_QUERY, WEBSITE_AUTH, (err, res, body) => {
+        if(!config.postIndividualDevices && !config.postDeviceSummary)
+        {
+            return resolve(true);
+        }
+        else
+        {

```

Comment:
```suggestion
        if(!config.postIndividualDevices && !config.postDeviceSummary)
        {
            return resolve(true);
        }
```
Remove unnecessary `else` statement